src/modelmanager.cpp

```diff
@@ -767,11 +761,18 @@ Status ModelManager::loadConfig(const std::string& jsonFilename) {
 
         if (!status.ok()) {
             IF_ERROR_NOT_OCCURRED_EARLIER_THEN_SET_FIRST_ERROR(status);
```

Comment:
Why do we need this macro here if we return right away anyway? This code could be simplified and moved to line 753.

```
if (!this->metricConfigLoadedOnce) {
  auto status = loadMetricsConfig(configJson);
  if (!status.ok() {
    return status;
  }
}
```

Reply:
Done.